### PR TITLE
Suggested Templates for Issues

### PR DESCRIPTION
**Pull request by jjhursey**
Three templates proposed (user can always open a non-templated Issue if necessary):
 *  `Discussion Topic (RFC)` for general topics and RFC level discussions. These are the deeper discussions.
 *  `Questions & Clarifications` for questions and clarifications. Meant to be lightweight.
 *  `Use Case` for use case discussions.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/use-case.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+A brief description of the use case.
+
+### Use Case Details
+
+A place for a longer description of your use case.
+Please also describe any alternative approaches that you have considered.
+
+### References
+
+List any prototypes, similar projects, papers, etc. that provide background and context for this use case. If the use case requires support from a specific resource
```
Comment (gvallee) [inline .github/ISSUE_TEMPLATE/use-case.md]:
```suggestion
List any prototypes, similar projects, papers, etc. that provide background and context for this use case. If the use case requires support from a specific resource.
```

**Comment by jjhursey** [comment]
A PR template for that is a good idea. For typos and grammar items I was thinking about having a general "Document Cleanup" Issue that the PR can reference (so they can skip that part of the process), then have the PR template automatically link back to that issue. This PR template can be lighter than a more general RFC PR template.

**Comment by jjhursey** [comment]
PR templates in PR #188

**Comment by jjhursey** [comment]
No objection in May 17 meeting. Merged

**Pull request merged**